__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+process.env.NODE_ENV = "test";
+
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => seed(testData));
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const topics = res.body;
+        expect(topics).toBeInstanceOf(Array);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+});
+
+describe("Error handling", () => {
+  test("404: responds with 400 error when received a request with invalid route(s)", () => {
+    return request(app)
+      .get("/")
```

Comment:
At the moment, this will only return a 404 if someone literally goes to the endpoint of "/", but we want it to return 404 if the user goes to ANY endpoint that we haven't created. How can we change this to be more generic (CLUE: you're one character away.)